src/components/EasyPill/EasyPill.tsx

```diff
@@ -1,13 +1,36 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
 import Media from '../Media';
 import EasyDropdown from '../EasyDropdown';
 import Icon from '../Icon';
 import Pill from '../Pill';
+import styles from './EasyPill.module.css';
 
-const EasyPillDropdown = ({ actions, onDelete }) => {
-  if (!actions.length) {
+type EasyPillActions = {
+  children?: React.ReactNode;
+  label: string;
+  onClick: () => void;
```

Comment:
This click event should be super permissive, since it's not used at all in this component, just passed down to a child.
I think the built-in `Function` type would work.
Then, any type checking can happen in the `EasyDropdown` component that receives the prop.

Reply:
Just for clarity, do you mean change `onClick: () => void;` to `onClick: Function;`?